lib/matplotlib/tests/test_axes.py

```diff
@@ -962,6 +962,18 @@ def test_hexbin_extent():
     ax.hexbin("x", "y", extent=[.1, .3, .6, .7], data=data)
 
 
+def test_hexbin_bad_extents():
+    fig, ax = plt.subplots()
+    data = (np.arange(2000) / 2000).reshape((2, 1000))
```

Comment:
Since this isn't going to be used for anything, we can probably shrink it a bit:
```suggestion
    data = (np.arange(20) / 20).reshape((2, 10))
```